### ISSUE TITLE
JSON stringifier / parser 

### ISSUE THREAD

**Issue by jlarmstrongiv** [DevUtilsApp/DevUtils-app#61]
Often times, whether it’s debugging requests or setting up json configs, I kept wishing I had this little utility.  

Related #48.  But in addition to a parser, also a stringifier

**Comment by trungdq88** [comment]
Hi there 👋

Can you elaborate on what would a "JSON stringifier" do or provide an example input/output? Thanks!

**Comment by jlarmstrongiv** [comment]
Sure

For example, I’m using a plugin called [headwind](https://github.com/heybourn/headwind) that allows for custom regex configuration.  I need to write the regex, and then stringify it.  

Input:
```
(?:\b(?:class|className|tw)?\.?\w?\s*=*\s*{?[\"\'\`]([_a-zA-Z0-9\s\-\:/]+)[\"\'\`]}?)
```
Output:
```
(?:\\b(?:class|className|tw)?\\.?\\w?\\s*=*\\s*{?[\\\"\\'\\`]([_a-zA-Z0-9\\s\\-\\:/]+)[\\\"\\'\\`]}?)
```

I also stringify when debugging requests with postman or insomnia, as I send JSON payloads a lot.  

The input can be a string, boolean, number, array, or object from JavaScript (as well as special type like undefined, null, regex, dates, and more too).  I’ve been using tools like:
- http://json.parser.online.fr/
- https://onlinetexttools.com/json-stringify-text

But it’d be great to have it in DevUtils 😄

**Comment by trungdq88** [comment]
Ah I see, thanks for elaborating.

For stringifier, if I understand it correctly, it means you want to escape the reserved character in a JSON string, right?

Do you think the existing tool "Backslash Escape/Unescape" can do it? For example:

![image](https://user-images.githubusercontent.com/4214509/107135924-609abe00-6931-11eb-81a6-75f27dc2abe4.png)

Can you test it out with a few cases and let me know if it gives the desired output? I'll consider updating the tool name/description to make it more friendly (maybe add a "stringifier" keyword to the description).

As for JSON Parser, the existing tool also allow you to see if there is any syntax error. Screenshot:

![image](https://user-images.githubusercontent.com/4214509/107135970-cedf8080-6931-11eb-9fd2-5e3f39053ee6.png)

Is there a more specific case you are looking to use that the current JSON Format/Validate tool doesn't support?

**Comment by jlarmstrongiv** [comment]
Not quite, the main goal is converting the aforementioned JSON <=> Javascript.  So both a stringifier and a parser.

The datatypes are normally objects and arrays, but also strings, booleans, numbers, null, and others.

The Backslash Escape/Unescape also doesn’t handle edge cases like converting undefined to null.  It’s also soooooo much more convenient to just copy pasta my entire clipboard rather than find and convert specific lines with the Backslash Escape/Unescape tool (when working with requests, for example).  Sure, the JSON Format/Validate tool will tell me that a JavaScript object is not JSON, but that’s not nearly as helpful as having a JavaScript to JSON converter actually convert it for me.

**Comment by trungdq88** [comment]
I see. This is well noted :) Thanks for the suggestion!
